Fix U-Boot build with GCC 14: enum/int mismatch in cmd_process

--- a/include/command.h
+++ b/include/command.h
@@ -231,7 +231,7 @@
  * @return 0 if the command succeeded, 1 if it failed
  */
-int cmd_process(int flag, int argc, char *const argv[], int *repeatable,
+enum command_ret_t cmd_process(int flag, int argc, char *const argv[], int *repeatable,
 		unsigned long *ticks);
 
 void fixup_cmdtable(struct cmd_tbl *cmdtp, int size);
